@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -837,6 +845,64 @@ export class VisibleTileSet {
         tile.dispose();
     }
 
+    /**
+     * Disposes of tiles that overlap twice.
+     **/
+    private checkDuplicateFullyCoveringTiles(dataSource: DataSource, tile: Tile) {
```

Comment:
The name indicates there is no side effect, but there is. So `removeDuplicateFullyCobveringTiles` would be a better name